### ISSUE TITLE
Quickstart broken due to CentOS7 EOL

### ISSUE THREAD

**Issue by keith-ratcliffe** [NationalSecurityAgency/datawave#2454]
During yum updates in the docker build...
```
 Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
       curl#6 - "Could not resolve host: mirrorlist.centos.org; Unknown error"
```

**Comment by keith-ratcliffe** [comment]
Temporary fix, until upgrade to a non-EOL image is complete: [d41e886](https://github.com/NationalSecurityAgency/datawave/commit/d41e8861aecab19d5010292cbaa2a3916bc09475)

**Comment by keith-ratcliffe** [comment]
Quickstart fixed by [d1b8678](https://github.com/NationalSecurityAgency/datawave/commit/d1b8678f8ea6dd89705c741632940312f01f4fba)

Opening new issue for the webservice image
